app/models/shipit/stack.rb

```diff
@@ -80,6 +80,14 @@ def trigger_deploy(until_commit, user, env: nil)
       deploy
     end
 
+    def trigger_continuous_deploy!
+      return unless deployable?
+      if commit = last_deployable_commit
+        return if commit.deployed?
+        trigger_deploy(commit, commit.committer)
```

Comment:
I'm not super familiar with Ruby style, but if this method has a `!` because it modifies internal state, I would assume `trigger_deploy` would also need to be suffixed with a `!`.